examples/basic_room/index.ts

```diff
@@ -29,7 +33,7 @@ console.log('connected to room', room);
 
 // read relevant metadata from wav file
 // this example assumes valid encoding little-endian
-const sample = readFileSync(join(import.meta.dirname, '../speex.wav'));
+const sample = readFileSync(join(process.cwd(), './speex.wav'));
```

Comment:
`import.meta.dirname` is in RC status and only available in NodeJS 20.11 and up.